lighthouse-core/audits/seo/canonical.js

```diff
@@ -90,6 +90,7 @@ class Canonical extends Audit {
           });
 
         canonicals = canonicals.concat(artifacts.Canonical);
+        canonicals = Array.from(new Set(canonicals));
```

Comment:
maybe add comment with link to discussion? Looks like some of it happened out of band, but https://github.com/GoogleChrome/lighthouse/issues/3178#issuecomment-381181762 is at least a start if you're wondering why it's deduping here :)

Reply:
👍 comment with explanation and link added